student.html

```diff
@@ -0,0 +1,5 @@
+---
+layout: home
+title: Student
+---
+{% include profiles.html content=page.name %}
```

Comment:
Nice and simple. Also leaves room to iterate future categories on here too, for example if we want to add a "Friends" section for folks not formally affiliated to RIT but who hang around the FOSS@RIT community!

Reply:
I didnt think of that, but yes, this design would support that